turbo/stages/headerdownload/header_algos.go

```diff
@@ -688,10 +690,16 @@ func (hd *HeaderDownload) InsertHeaders(hf func(header *types.Header, hash commo
 			continue
 		}
 
-		// Check if transition to proof-of-stake happened
-		if err := hf(link.header, link.hash, link.blockHeight, terminalTotalDifficulty); err != nil {
+		isTrans, err := hf(link.header, link.hash, link.blockHeight, terminalTotalDifficulty)
```

Comment:
What does "hf" mean? Please rename to something clearer.

Reply:
i did not name it, it was there before, i think it stands for header function

Reply:
it is the header insertion function in Erigon

Reply:
OK, fair enough.